packages/display/src/DisplayObject.js

```diff
@@ -266,20 +277,6 @@ export default class DisplayObject extends EventEmitter
         if (!skipUpdate)
         {
             this._recursivePostUpdateTransform();
```

Comment:
Don't you need to call `this.safeUpdateTransform();` here?

Reply:
Nope, its not required for toLocal/toGlobal. there was single displayObjectUpdateTransform, without children, but its called from _recursivePostUpdateTransform anyway!

Reply:
Cool, just making sure